### PR TITLE
treewide: remove maintainer goibhniu

### PR DESCRIPTION
**Pull request by tie**
## Description of changes

Inactive in Nixpkgs since 2018 (and a single PR in 2022).
https://github.com/NixOS/nixpkgs/issues?q=author%3Acillianderoiste

Pinging @cillianderoiste (aka `goibhniu` in `maintainer-list.nix`).

See https://github.com/NixOS/nixpkgs/issues/290642

According to `grep` from #327779, this change leaves 91 packages without maintainer.

<details>
<summary>List of orphaned packages after this change</summary>

```
pkgs/applications/audio/a2jmidid/default.nix
pkgs/applications/audio/ams-lv2/default.nix
pkgs/applications/audio/bristol/default.nix
pkgs/applications/audio/calf/default.nix
pkgs/applications/audio/distrho/default.nix
pkgs/applications/audio/drumkv1/default.nix
pkgs/applications/audio/gigedit/default.nix
pkgs/applications/audio/jack-oscrolloscope/default.nix
pkgs/applications/audio/jalv/default.nix
pkgs/applications/audio/lash/default.nix
pkgs/applications/audio/linuxsampler/default.nix
pkgs/applications/audio/mda-lv2/default.nix
pkgs/applications/audio/mhwaveedit/default.nix
pkgs/applications/audio/mid2key/default.nix
pkgs/applications/audio/petrifoo/default.nix
pkgs/applications/audio/qjackctl/default.nix
pkgs/applications/audio/qsampler/default.nix
pkgs/applications/audio/qsynth/default.nix
pkgs/applications/audio/rakarrack/default.nix
pkgs/applications/audio/samplv1/default.nix
pkgs/applications/audio/seq24/default.nix
pkgs/applications/audio/setbfree/default.nix
pkgs/applications/audio/swh-lv2/default.nix
pkgs/applications/audio/synthv1/default.nix
pkgs/applications/audio/vkeybd/default.nix
pkgs/applications/audio/xsynth-dssi/default.nix
pkgs/applications/audio/yoshimi/default.nix
pkgs/applications/graphics/synfigstudio/default.nix
pkgs/applications/misc/artha/default.nix
pkgs/applications/video/simplescreenrecorder/default.nix
pkgs/by-name/qt/qtractor/package.nix
pkgs/development/libraries/aqbanking/default.nix
pkgs/development/libraries/aqbanking/gwenhywfar.nix
pkgs/development/libraries/audio/libsmf/default.nix
pkgs/development/libraries/audio/lilv/default.nix
pkgs/development/libraries/audio/lv2/default.nix
pkgs/development/libraries/audio/lvtk/default.nix
pkgs/development/libraries/audio/qm-dsp/default.nix
pkgs/development/libraries/audio/sratom/default.nix
pkgs/development/libraries/audio/suil/default.nix
pkgs/development/libraries/dbus-cplusplus/default.nix
pkgs/development/libraries/frei0r/default.nix
pkgs/development/libraries/kissfft/default.nix
pkgs/development/libraries/libconfig/default.nix
pkgs/development/libraries/libepoxy/default.nix
pkgs/development/libraries/libfakekey/default.nix
pkgs/development/libraries/libgig/default.nix
pkgs/development/libraries/liblscp/default.nix
pkgs/development/libraries/minixml/default.nix
pkgs/development/libraries/mlt/default.nix
pkgs/development/libraries/movit/default.nix
pkgs/development/libraries/opencolorio/1.x.nix
pkgs/development/libraries/openimageio/default.nix
pkgs/development/libraries/robin-map/default.nix
pkgs/development/libraries/serd/default.nix
pkgs/development/libraries/sord/default.nix
pkgs/development/python-modules/evdev/default.nix
pkgs/development/python-modules/limnoria/default.nix
pkgs/development/python-modules/pathtools/default.nix
pkgs/development/python-modules/rope/default.nix
pkgs/development/python-modules/svg-path/default.nix
pkgs/development/python-modules/virtualenv/default.nix
pkgs/development/python-modules/watchdog/default.nix
pkgs/development/python-modules/zbaemon/default.nix
pkgs/development/python-modules/zc-lockfile/default.nix
pkgs/development/python-modules/zconfig/default.nix
pkgs/development/python-modules/zdaemon/default.nix
pkgs/development/python-modules/zodb/default.nix
pkgs/development/python-modules/zope-component/default.nix
pkgs/development/python-modules/zope-configuration/default.nix
pkgs/development/python-modules/zope-contenttype/default.nix
pkgs/development/python-modules/zope-dottedname/default.nix
pkgs/development/python-modules/zope-event/default.nix
pkgs/development/python-modules/zope-exceptions/default.nix
pkgs/development/python-modules/zope-filerepresentation/default.nix
pkgs/development/python-modules/zope-i18nmessageid/default.nix
pkgs/development/python-modules/zope-interface/default.nix
pkgs/development/python-modules/zope-lifecycleevent/default.nix
pkgs/development/python-modules/zope-location/default.nix
pkgs/development/python-modules/zope-proxy/default.nix
pkgs/development/python-modules/zope-schema/default.nix
pkgs/development/python-modules/zope-size/default.nix
pkgs/development/python-modules/zope-testing/default.nix
pkgs/development/python-modules/zope-testrunner/default.nix
pkgs/misc/jackaudio/default.nix
pkgs/os-specific/linux/xf86-video-nested/default.nix
pkgs/servers/http/yaws/default.nix
pkgs/tools/misc/ttfautohint/default.nix
pkgs/tools/security/aespipe/default.nix
pkgs/tools/text/podiff/default.nix
pkgs/tools/text/uni2ascii/default.nix
```
</details>

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by cillianderoiste** [comment]
Thanks for cleaning that up!

File: pkgs/applications/audio/lmms/default.nix [hunk 1]
```diff
@@ -52,6 +52,6 @@ mkDerivation rec {
     homepage = "https://lmms.io";
     license = licenses.gpl2Plus;
     platforms = [ "x86_64-linux" "i686-linux" ];
-    maintainers = with maintainers; [ goibhniu ];
+    maintainers = with maintainers; [ yana ];
```
Comment (jopejoe1) [inline pkgs/applications/audio/lmms/default.nix:55]: Of-borg can't find a maintainer under the name yana
```suggestion
    maintainers = with maintainers; [  ];
```

**Review by jopejoe1 — DISMISSED** [review]

File: pkgs/applications/audio/lmms/default.nix (hunk 1 above)
Comment (tie) [reply pkgs/applications/audio/lmms/default.nix:55]: See 39ddcbfc694c254f1af2dbd6519c7f0bf7bba055

Slipped when rebasing.

Comment (tie) [reply pkgs/applications/audio/lmms/default.nix]: Fixed.

**Review by drupol — APPROVED** [review]

**Pull request merged**